.eslintrc.js

```diff
@@ -14,10 +14,13 @@ module.exports = {
             rules: {
                 // Allow JSX to be written in any file ignoring the extension type
                 'react/jsx-filename-extension': 'off',
+                'rulesdir/prefer-underscore-method': 'off',
```

Comment:
Remove the same line below (line 53):
```ts
 'rulesdir/prefer-underscore-method': 'off',
```

Reply:
Why should I remove it, without it some eslint errors appear

Reply:
At the time I reviewed the PR this rule was duplicated, now it's fine 👍

Reply:
Actually, you have lint errors in ts files now, turns out both rules are needed 🤷‍♂️